common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -17,6 +17,8 @@ namespace beerocks {
 
 #define BEEROCKS_CONTROLLER "beerocks_controller"
 #define BEEROCKS_AGENT "beerocks_agent"
+#define BEEROCKS_FRONTHAUL "beerocks_fronthaul"
+#define BEEROCKS_AP_MANAGER "beerocks_ap_manager"
```

Comment:
This will be removed in the future?

Reply:
No. The BEEROCKS_FRONTHAUL is for pid file touch, and BEEROCKS_AP_MANAGER/MONITOR is for the logger.